packages/nodes-base/nodes/Kafka/KafkaTrigger.node.ts

```diff
@@ -56,6 +58,28 @@ export class KafkaTrigger implements INodeType {
 				placeholder: 'n8n-kafka',
 				description: 'ID of the consumer group.',
 			},
+			{
+				displayName: 'Use Schema Registry',
+				name: 'useSchemaRegistry',
+				type: 'boolean',
+				default: false,
+				description: 'Use Apache Avro serialization format and Confluent\' wire formats.',
```

Comment:
This description can be simplified to avoid technical words. (the same in the Kafka node)

Reply:
done